sdk/monitor/monitor-opentelemetry-exporter/package.json

```diff
@@ -67,14 +67,8 @@
     "@typescript-eslint/parser": "^2.0.0",
     "c8": "^7.2.0",
     "eslint": "^6.1.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
     "execa": "^3.3.0",
-    "eslint-plugin-no-null": "^1.0.2",
-    "eslint-plugin-no-only-tests": "^2.3.0",
-    "eslint-plugin-promise": "^4.1.1",
```

Comment:
I remember I needed to install these because they were peer dependencies of the azure-sdk eslint plugin. Else eslint failed to run. Are these included with the azure eslint plugin now?

Reply:
some of them are indeed peer deps for our plugin (see [here](https://github.com/Azure/azure-sdk-for-js/blob/c5f9aa8887a4010ce6e753f10b10ad3ead9cb0de/common/tools/eslint-plugin-azure-sdk/package.json#L59)). However, `eslint 6.8` runs fine on my machine without them being deps here. Would you mind cloning this branch and trying it?

Reply:
@markwolff I created this issue to investigate how those deps are being pulled https://github.com/Azure/azure-sdk-for-js/issues/11667. For now, I would like to merge this PR to remove the unnecessary dependencies and we can always add them back if it turns out they are needed.